aemiers/rancid-tomatillos
# Fix/focus and hover

## Pull Request

### Description

* Is this a feature or a fix?
Fix. 
* Where should the reviewer start?
Review the SASS files for MovieCard to see how I matched the hover and focus states.
* What functionalities do these changes affect? Why were these changes made? What was the motivation behind these changes?
The hover states and focus states weren't matching. 
* What outside features or research did you use to implement this fix?
I had to search for ways to target children elements when parents where had focus, which lead me to this stackoverflow. https://stackoverflow.com/questions/7217244/style-child-element-when-hover-on-parent 
* How should this be tested?
Pull it down, hover and focus over the movie details to confirm that it is working as expected. 

***

#### Please review considerations below:
